packages/api/src/generated/schema.graphql

```diff
@@ -1294,6 +1351,7 @@ type Query {
   article(format: String, slug: String!, username: String!): ArticleResult!
   articleSavingRequest(id: ID, url: String): ArticleSavingRequestResult!
   deviceTokens: DeviceTokensResult!
+  feeds(input: FeedsInput!): FeedsResult!
```

Comment:
Sorry, i realized maybe this should be named `following`

Reply:
@jacksonh, I think maybe I didn't make it very obvious but this API is to search an RSS feed by name or URL but to search following which we would use the same search API

Reply:
Some users want to search for an RSS feed by name so they won't need to memorise or fill in the complete URL when they subscribe to one